index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
-    <title>Portfolio Setup</title>
+    <title>Portfolio</title>
```

Comment:
- [x] Great work on working on the page title, however, make sure it adequately and briefly describes the content of the page for better accessibility on the internet by Search Engines.

```suggestion
    <title>Abu's Portfolio</title>
```

Reply:
- [ ] Or even `Abu FullStack Developer`